lib/parsers/csv_layout.rb

```diff
@@ -77,8 +77,8 @@ def convert_to_location(str)
       return nil
     end
 
-    def validate_tube_barcode_format(barcode)
-      unless valid_fluidx_barcode?(barcode) || no_read_barcode?(barcode)
+    def validate_barcode_format(barcode)
+      unless valid_barcode?(barcode) || no_read_barcode?(barcode)
         @errors.push(:msg => "Invalid Fluidx tube barcode format #{barcode}")
```

Comment:
The error msg here refers specifically to "Fluidx", this seems wrong.

It feels like validating the barcode should be a separate object/function belonging to the object parsing the csv.
Then any error messages could be specific to the barcode itself.